QC/wdl/workflows/SV_assembly.wdl

```diff
@@ -8,6 +9,15 @@ import "../tasks/jasmine.wdl" as runJasmine
 workflow sv_assembly{
     
     input{
+        File ParlInputBam
+        File ParlRefGenome
```

Comment:
I think you have some replication here:
ParlRefGenome should be the same as GenomeIn

Also, 
ReadsIn seems replicated as well.